eng/pipelines/templates/stages/cosmos-sdk-client.yml

```diff
@@ -86,6 +86,7 @@ stages:
         ServiceDirectory: cosmos
         Artifacts: ${{ parameters.Artifacts }}
         AdditionalModules: ${{ parameters.AdditionalModules }}
+        TimeoutInMinutes: 90
```

Comment:
Please include a comment explaining that the timeout is extended because the Cosmos Emulator takes a very long time to download and link this bug: https://github.com/Azure/azure-sdk-for-java/issues/12970

A glance through download times for Cosmos DB in Python and JS hosts shows that the download step is taking a very long time compared to Java. It's not clear why this is the case. The bug above is tracking this.

Reply:
Thanks for creating the issue @danieljurek 
I have added the comment in this PR .